### ISSUE TITLE
keep original columns for clean_data

### ISSUE THREAD

**Issue by dirkschumacher** [reconhub/linelist#1]
Having a magical function that does everything is great, but I can imagine that keeping the original values of the columns helps to trust the transformation.

E.g. the values before the transformations could be kept in the resulting data frame with an added suffix `date_of_onset_original` or something

**Comment by zkamvar** [comment]
IIRC, @thibautjombart is a bit opposed to this concept for the fact that the user can do:

```r
old_data <- the_data
the_data <- linelist::clean_data(the_data)
```

**Comment by dirkschumacher** [comment]
Keeping the original and modified data together helps spot errors. Especially if you use a magic function like "clean_data" that might behave differently with future releases of linelist. What about adding a parameter with the default to include the original columns?

**Comment by zkamvar** [comment]
I think that's a good idea! Plus, there could be a function that uses diffObj to compare the cleaned and original columns.

**Comment by thibautjombart** [comment]
I agree, as an additional argument. I would add the columns so that the original and 'cleaned' variables are next to each other:
```
toto   toto_clean  tata   tata_clean ...
```
